### DISCUSSION TITLE
Flash task bar button on bell when inactive

### DISCUSSION THREAD

**Discussion by Ennea** [wezterm/wezterm#2612] [Ideas]
First off, thank you so much for wezterm, Wez (and contributors)! Such a joy to use, I love it :)

Onto my (hopefully simple) request/proposal: I'd like to have wezterm's task bar/panel button flash when it receives a bell while it's currently inactive.

**Comment by wez** [comment]
https://github.com/wez/wezterm/pull/1636 is a PR that started to support this on X11 systems, but also has some discussion about supporting it on the other systems

**Reply by Ennea** [reply]
Ah, I was not able to find that. Looks interesting! I'll take a closer look and see if I may be able to help :)
